nautobot_floor_plan/templates/nautobot_floor_plan/inc/floorplan_svg.html

```diff
@@ -1,8 +1,27 @@
 {% load buttons %}
 
 {% if floor_plan %}
-    <h4>{{ floor_plan }} {% edit_button floor_plan %}</h4>
-    Use shift+scroll wheel to zoom in or out. Click and drag to scroll.
+    <h4>{{ floor_plan }}</h4>
+    <div class="row">
+        <div class="col-md-1"></div>
+        <div class="col-md-8">
+            Use shift+scroll wheel to zoom in or out.
+            Click and drag to scroll.<br>
+            Add a tile by clicking the plus symbol on the grid OR by using the "Add Tile" button.
+        </div>
+        <div class="col-md-2">
+            <div class="btn-group btn-group-sm btn-group-justified" role="group">
```

Comment:
We don't *in general* use btn-group for separate/loosely related buttons in the UI - while it looks nice enough, my vote would be to have these as separate side-by-side buttons rather than a grouped button.

You might also want to explore whether defining these buttons in a `{% block extra_buttons %}` (https://docs.nautobot.com/projects/core/en/stable/development/core/templates/#object-detail), or alternatively a `<div class="pull-right">` instead of a separate column, gives a more self-consistent placement on the page.

Reply:
Thanks for the documentation link for the object-detail! I tried adding it to extra_buttons but it looks a little too busy up there with these extra buttons. Good call out with the pull-right though. Code change incoming